python/lsst/obs/monocam/monocamIsrTask.py

```diff
@@ -111,7 +113,7 @@ def run(self, ccdExposure, bias=None, dark=None, flat=None, defects=None, fringe
         if self.config.doFringe and self.config.fringeAfterFlat:
             self.fringe.run(ccdExposure, **fringes.getDict())
 
-        ccdExposure.getCalib().setFluxMag0(self.config.fluxMag0T1 * ccdExposure.getCalib().getExptime())
+        ccdExposure.setPhotoCalib(lsst.afw.image.makePhotoCalibFromCalibZeroPoint(self.config.fluxMag0T1, 0))
```

Comment:
Why do you not need to take into account exposure time as it was before?

Reply:
There has not been an exposure time in the `Calib` object for years (see DM-5503), so the line as written won't even execute correctly. I don't know what the correct line would be, and there are no tests, so...